src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -143,6 +143,30 @@ case class SubmissionReport(
   notstarted: Seq[SubmissionValidationEntityInputs]
 )
 
+case class CallMetadata(
+  inputs: Map[String, String],
+  status: String,
+  outputs: Option[Map[String, String]],
+  start: Option[DateTime],
+  end: Option[DateTime],
+  jobid: Option[String],
+  rc: Option[Int],
```

Comment:
I've Voetized the name to `returnCode`.

On a non-Voet note I also added a field on `CallMetadata` called `backend` of type `Option[String]` to indicate the type of backend on which the call executed.